server/src/test/java/org/elasticsearch/search/aggregations/metrics/CardinalityAggregatorTests.java

```diff
@@ -181,25 +177,9 @@ private void testCase(Query query, CheckedConsumer<RandomIndexWriter, IOExceptio
         testCase(aggregationBuilder, query, buildIndex, verify, fieldType);
     }
 
-    private void testCase(CardinalityAggregationBuilder aggregationBuilder, Query query,
+    protected void testCase(CardinalityAggregationBuilder aggregationBuilder, Query query,
```

Comment:
Maybe rename these so they don't "hide" the superclass method and require the casts?